autogen/oai/client.py

```diff
@@ -161,6 +169,14 @@ def create(self, params: Dict[str, Any]) -> ChatCompletion:
 
         completions: Completions = self._oai_client.chat.completions if "messages" in params else self._oai_client.completions  # type: ignore [attr-defined]
         # If streaming is enabled and has messages, then iterate over the chunks of the response.
+
+        _is_vision = is_vision_model(params.get("model", None))
+        _is_vision |= params.get("vision_model", False)
+        if _is_vision and "messages" in params:
+            if format_message_contents_with_images is None:
+                raise ImportError("Please install PIL to use the vision model.")
```

Comment:
Is PIL always required? It doesn't sound right.

Reply:
If the input is already an URI, then we don't need PIL.

IF the input is a file location or URL, then PIL is needed to load the image data.

Reply:
Based on what you said, this error shouldn't be raised here. Maybe I misunderstood it.